ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -130,4 +134,21 @@ object LedgerApiTestTool {
     }
   }
 
+  private[this] def newLedgerSuiteRunner(
+      config: Config,
+      suites: Iterable[LedgerSession => LedgerTestSuite],
+      concurrencyOverride: Int = -1): LedgerTestSuiteRunner =
```

Comment:
Please use an `Option[Int]` here. This isn't C. 😛

Reply:
Don't really see an advantage here (more complex to call, possibly additional GC pressure) but OK